### PR TITLE
Publish Crowdin monthly May

### PR DESCRIPTION
**Pull request by Julievrz**
Publish Crowdin monthly May 2022

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Julievrz

**Comment by matks** [comment]
To be published on 7th of June as I can see from the publish date

File: news/_posts/translation/do-you-speak-prestashop-may-2022.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – May 2022 edition"
+subtitle: "Where you'll learn that PrestaShop 1.7.8 is now fully available in Portuguese BR "
```
Comment (matks) [inline news/_posts/translation/do-you-speak-prestashop-may-2022.md]:
```suggestion
subtitle: "Where you'll learn that PrestaShop 1.7.8 is now fully available in Portuguese BR"
```
Un-needed white space

Comment (Julievrz) [reply news/_posts/translation/do-you-speak-prestashop-may-2022.md]: Thank you!

File: news/_posts/translation/do-you-speak-prestashop-may-2022.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: "Do you speak PrestaShop? – May 2022 edition"
+subtitle: "Where you'll learn that PrestaShop 1.7.8 is now fully available in Portuguese BR"
+date: 2022-06-07
```
Comment (matks) [inline news/_posts/translation/do-you-speak-prestashop-may-2022.md]:
```suggestion
date: 2022-06-07 09:00:00
```

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Publication issue again 🙄

**Pull request merged**